theme/stencil/src/components/hubspot-form/hubspot-form.tsx

```diff
@@ -148,8 +148,9 @@ export class HubspotForm {
                 utmSource: utmData.source,
                 utmMedium: utmData.medium,
             };
-
-            analytics.track("form-submission", submissionData);
+            if(emailAddress !== ""){ // Don't track empty email addresses
```

Comment:
This works, but would suggest putting this check higher up, since the work of the function is about submitting:

```
if (analyticsAvailable && emailAddress !== "") {
    // ...
```

Reply:
Fixed!! Thanks